config/common/culture/cultures.cwt

```diff
@@ -0,0 +1,163 @@
+types = {
+    ## type_key_filter <> mercenary_names
+    type[culture] = {
+        path = "game/common/culture/cultures"
+        skip_root_key = any
+    }
+    type[culture_group] = {
+        path = "game/common/culture/cultures"
+    }
+}
+culture_group = {
+    # TODO: Make graphical cultures somehow matter.
+    ## cardinality = 0..1
+    ### List of graphical cultures used for coat of arms, buildings, clothings and units. It's possible to give more than one from each type, then all of them will be used.
+    graphical_cultures = single_alias_right[scalar_list]
+    ## cardinality = 0..1
+    ### List of possible mercenary names and optionally coats of arms.
+    mercenary_names = single_alias_right[mercenary_list]
+    ## cardinality = 1..inf
+    <culture> = {
+        ### The color of the culture, used e.g. on the map.
+        color = Color
+
+        ## cardinality = 0..1
+        ### List of graphical cultures used for coat of arms, buildings, clothings and units. It's possible to give more than one from each type, then all of them will be used.
+        graphical_cultures = single_alias_right[scalar_list]
+        ## cardinality = 0..1
+        ### List of possible mercenary names and optionally coats of arms.
+        mercenary_names = single_alias_right[mercenary_list]
+        
+        ## cardinality = 0..1
+        ### List of possible cadet dynasty names.
+        cadet_dynasty_names = single_alias_right[dynasty_name_list]
+        ## cardinality = 1
+        ### List of possible dynasty names.
+        dynasty_names = single_alias_right[dynasty_name_list]
+
+        ## cardinality = 0..1
+        ### When generating a dynasty name based on a title, add this prefix.
+        dynasty_of_location_prefix = localisation
+        ## cardinality = 0..1
+        ### When generating a dynasty name for a bastard, add this prefix.
+        bastard_dynasty_prefix = localisation
+
+        ## cardinality = 0..1
+        founder_named_dynasties = bool
+        ## cardinality = 0..1
+        dynasty_title_names = bool
+        ## cardinality = 0..1
+        dynasty_name_first = bool
+
+        male_names = single_alias_right[first_name_list]
+        male_names = single_alias_right[first_name_weighted_list]
+
+        female_names = single_alias_right[first_name_list]
+        female_names = single_alias_right[first_name_weighted_list]
+
+        ## cardinality = 0..1
+        ### Chance of male children being named after their paternal grandfather. Sum of these three values must not exceed 100.
+		pat_grf_name_chance = int
```

Comment:
Are we sure this and the following fields are int and not `int_value_field`?

Reply:
Tried it with a value field, throws errors.